### PR TITLE
OpenConfig: add required sequence-id field

### PR DESCRIPTION
**Pull request by XioNoX**
Fixes issue #309

### CODE REVIEW

File: aerleon/lib/openconfig.py [hunk 1]
```diff
@@ -262,6 +262,7 @@ def _TranslatePolicy(self, pol: Policy, exp_info: int) -> None:
                     t = Term(term, term_af)
                     for rule in t.ConvertToDict():
                         total_rule_count += 1
+                        rule['sequence-id'] = total_rule_count * 5
```
Comment (ankenyr) [inline aerleon/lib/openconfig.py:265]: Why is this multiplied by 5? I am looking at the documentation at https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-acl.html and do not see anything that explains that.

Comment (XioNoX) [reply aerleon/lib/openconfig.py:265]: I put the reason in the commit message.
It's sometimes (now less than before thanks to automation) useful to be able to manually add a rule in between two existing rules directly though the device's CLI. Like for testing or as incident response.

Comment (ankenyr) [reply aerleon/lib/openconfig.py:265]: Ahh so for future reference, we squash commits so I don't believe those messages get preserved. You should add information like this into the initial PR.

**Comment by ankenyr** [comment]
Tests are failing here. I think it is because the order of the output is wrong.

**Review by ankenyr — APPROVED** [review]

**Pull request merged**